docs/docusaurus.config.js

```diff
@@ -80,10 +80,10 @@ const config = {
             ],
           },
           {
-            title: "Compute@Edge",
+            title: "Compute",
             items: [
               {
-                label: "Learn about Compute@Edge",
+                label: "Learn about Compute",
```

Comment:
I feel like this could use the "Fastly" prefix...

```suggestion
                label: "Learn about Fastly Compute",
```